src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -941,10 +941,9 @@ static void G(object o)
     }
 }";
             var comp = CreateCompilation(source, parseOptions: TestOptions.Regular8, options: TestOptions.ReleaseModule);
-            comp.VerifyEmitDiagnostics(
-                // (9,14): error CS0518: Predefined type 'System.Runtime.CompilerServices.NullableAttribute' is not defined or imported
-                //         F(() =>
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "=>").WithArguments("System.Runtime.CompilerServices.NullableAttribute").WithLocation(9, 14));
+            // No missing attribute warning since the nullable state of the lambda
+            // return type was inferred in flow analysis, not from initial binding.
+            comp.VerifyEmitDiagnostics();
```

Comment:
I didn't understand the comment.
- Do we still end up emitting using the attribute somewhere?
- How can this attribute be missing? Isn't this an attribute we can generate/embed when needed?

Reply:
The lambda signature is emitted without a `[Nullable]` attribute because the return type nullability is inferred rather than explicit. Updated the comment.

---
In reply to: [192622649](https://github.com/dotnet/roslyn/pull/27283#discussion_r192622649) [](ancestors = 192622649)

Reply:
Sorry. Still didn't understand. I'll stop by.

---
In reply to: [192774627](https://github.com/dotnet/roslyn/pull/27283#discussion_r192774627) [](ancestors = 192774627,192622649)